### PR TITLE
Add cmake example to the documentation

### PR DESCRIPTION
**Pull request by SimeonEhrig**
The PR address the Issue #961. The example explains, how you add Alpaka to the `CMakelists.txt` of your project. It really minimalistic and it includes the `helloWorld.cpp` example file. Feel free to give hints, if something is missing, not clear or looks confusing.

### CODE REVIEW

**Review by sbastrakov — DISMISSED** [review]

File: docs/source/usage/cmake_example.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.. highlight:: bash
+
+CMake Example
+=============
+
+You can integrate Alpaka in you project via ``find_package()`` in your ``CMakeLists.txt``. That requires, that you :doc:`install </install/instructions>` Alpaka. If you do not install Alpaka in a default path such ``/use/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```
Comment (BenjaminW3) [inline docs/source/usage/cmake_example.rst]:
```suggestion
You can integrate Alpaka in you project via ``find_package()`` in your ``CMakeLists.txt``. This requires, that you :doc:`install </install/instructions>` Alpaka. If you do not install Alpaka in a default path such as ``/use/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```

**Review by BenjaminW3 — DISMISSED** [review]

File: docs/source/usage/cmake_example.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.. highlight:: bash
+
+CMake Example
+=============
+
+You can integrate Alpaka in you project via ``find_package()`` in your ``CMakeLists.txt``. This requires, that you :doc:`install </install/instructions>` Alpaka. If you do not install Alpaka in a default path such as ``/use/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```
Comment (sbastrakov) [inline docs/source/usage/cmake_example.rst]:
```suggestion
You can integrate Alpaka in you project via ``find_package()`` into your ``CMakeLists.txt``. This requires, that you :doc:`install </install/instructions>` Alpaka. If you do not install Alpaka in a default path such as ``/use/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```

Comment (sbastrakov) [inline docs/source/usage/cmake_example.rst]: Not sure, shouldn't each sentence start from a new line?

Comment (sbastrakov) [inline docs/source/usage/cmake_example.rst]:
```suggestion
You can integrate Alpaka in you project via ``find_package()`` in your ``CMakeLists.txt``. This requires, that you :doc:`install </install/instructions>` Alpaka. If you do not install Alpaka in a default path such as ``/usr/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```

**Review by sbastrakov — CHANGES_REQUESTED** [review]

File: docs/source/usage/cmake_example.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.. highlight:: bash
+
+CMake Example
+=============
+
+You can integrate Alpaka in you project via ``find_package()`` in your ``CMakeLists.txt``. This requires, that you :doc:`install </install/instructions>` Alpaka. If you do not install Alpaka in a default path such as ``/usr/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```
Comment (BenjaminW3) [inline docs/source/usage/cmake_example.rst]:
```suggestion
You can integrate Alpaka in you project via ``find_package()`` into your ``CMakeLists.txt``.
This requires, that you :doc:`install </install/instructions>` Alpaka.
If you do not install Alpaka in a default path such as ``/usr/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```

**Comment by BenjaminW3** [comment]
@sbastrakov I updated the PR with your comments.

File: docs/source/usage/cmake_example.rst (hunk 2 above)
Comment (sbastrakov) [reply docs/source/usage/cmake_example.rst]: Sorry, i did not write the change clearly. `in your CMakeLists.txt` was fine and did not need a change. What needed a change was `You can integrate Alpaka in you project ` to be corrected as   `You can integrate Alpaka into your project` (so both a typo and a wrong preposition).

**Comment by sbastrakov** [comment]
Thanks, however it seems one change was done mistakenly, wrote in that thread.

File: docs/source/usage/cmake_example.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
+.. highlight:: bash
+
+CMake Example
+=============
+
+You can integrate Alpaka in you project via ``find_package()`` into your ``CMakeLists.txt``.
```
Comment (BenjaminW3) [inline docs/source/usage/cmake_example.rst]:
```suggestion
You can integrate Alpaka into your project via ``find_package()`` in your ``CMakeLists.txt``.
```

File: docs/source/usage/cmake_example.rst (hunk 2 above)
Comment (BenjaminW3) [reply docs/source/usage/cmake_example.rst]: Now it should be fixed.

**Review by sbastrakov — APPROVED** [review]

**Pull request merged**